tests/integration_tests/test_sequence_features.py

```diff
@@ -0,0 +1,350 @@
+
+        if enc_encoder == 'parallel_cnn':
+            number_parallel_cnn_layers = \
+                len(model.model.input_features[input_feature_name].encoder_obj.conv_layers)
+            hidden_size = input_features[0]['state_size'] * number_parallel_cnn_layers
+            assert encoder_out['encoder_output'].shape.as_list() == \
+                   [batch_size, seq_size, hidden_size]
+
+        elif enc_encoder == 'stacked_parallel_cnn':
+            number_parallel_cnn_layers = \
+                len(model.model.input_features[input_feature_name]\
+                    .encoder_obj.parallel_conv1d_stack\
+                    .stacked_parallel_layers[0])
+            hidden_size = input_features[0]['state_size'] * number_parallel_cnn_layers
+            assert encoder_out['encoder_output'].shape.as_list() == \
+                   [batch_size, seq_size, hidden_size]
+
+        elif enc_encoder == 'rnn':
+            assert encoder_out['encoder_output'].shape.as_list() == \
+                   [batch_size, seq_size, TEST_HIDDEN_SIZE]
+
+            assert 'encoder_output_state' in encoder_out
+            if enc_cell_type == 'lstm':
+                assert isinstance(encoder_out['encoder_output_state'], list)
+                assert encoder_out['encoder_output_state'][0].shape.as_list() == \
+                       [batch_size, TEST_HIDDEN_SIZE]
+                assert encoder_out['encoder_output_state'][1].shape.as_list() == \
+                       [batch_size, TEST_HIDDEN_SIZE]
+            else:
+                assert isinstance(encoder_out['encoder_output_state'], tf.Tensor)
+                assert encoder_out['encoder_output_state'].shape.as_list() == \
+                       [batch_size, TEST_HIDDEN_SIZE]
+
+        elif enc_encoder == 'cnnrnn':
+            assert encoder_out['encoder_output'].shape.as_list() == \
+                   [batch_size, 1, TEST_HIDDEN_SIZE]
+            assert 'encoder_output_state' in encoder_out
+
+            if enc_cell_type == 'lstm':
+                assert isinstance(encoder_out['encoder_output_state'], list)
+                assert encoder_out['encoder_output_state'][0].shape.as_list() \
+                       == [batch_size, TEST_HIDDEN_SIZE]
+                assert encoder_out['encoder_output_state'][1].shape.as_list() \
+                       == [batch_size, TEST_HIDDEN_SIZE]
+            else:
+                assert isinstance(encoder_out['encoder_output_state'], tf.Tensor)
+                assert encoder_out['encoder_output_state'].shape.as_list() \
+                       == [batch_size, TEST_HIDDEN_SIZE]
+
+        elif enc_encoder == 'stacked_cnn':
+            assert encoder_out['encoder_output'].shape.as_list() \
+                   == [batch_size, 1, TEST_HIDDEN_SIZE]
+
+        elif enc_encoder == 'transformer':
+            assert encoder_out['encoder_output'].shape.as_list() \
+                   == [batch_size, seq_size, 256]
```

Comment:
why here it's 256 instead of TEST_HIDDEN_SIZE ?

Reply:
The reason for 256.  The `sequence_feature()` does not set a value for the `hidden_size`, which is the parameter used in Transformer encoder.  This means the encoder will take the default value of 256.   All the other encoders use the `fc_size`/`state_size` to set the hidden state size.  These are specified in `sequence_feature()` and set to 8.   So I use TEST_HIDDEN_SIZE for the other encoders.  

For clarity I can add a constant like `TEST_TRANSFORMER_HIDDEN_SIZE = 256`.

Did this make sense?

Reply:
I think you could set the transformer hidden size and the embedding size both to be equal to TEST_HIDDEN_SIZE and that should address the iddue (and speed up the test involving the transformer ;) )

Reply:
updated `sequence_feature()` to set `hidden_size` to 8 and replaced hardcoded 256 with `TEST_HIDDEN_SIZE`